hapi/hedera-protobuf-java-api/src/main/proto/block/stream/block_item.proto

```diff
@@ -236,8 +236,8 @@ message BlockItem {
          *   <li>The block SHALL contain _exactly one_ `RecordFileItem`.</li>
          *   <li>The block SHALL NOT contain any item other than a
          *       `RecordFileItem`.</li>
-         *   <li>The content of the `RecordFileItem` MUST be validated using
-         *       the signature data and content provided within according to
+         *   <li>The content of the `RecordFileItem` CAN be validated using
+         *       the SignedRecordFileProof according to
```

Comment:
Not understanding `MUST` -> `CAN`. How will `RecordFileItem` be validated if not using `SignedRecordFileProof`?